ote/src/cljs/ote/views/service_search.cljs

```diff
@@ -97,12 +95,16 @@
    (when-not (empty? external-interface-links)
      (doall
        (map-indexed
-         (fn [i {::t-service/keys [external-interface format data-content] :as row}]
+         (fn [i {::t-service/keys [external-interface format data-content]
+                 :keys [url-ote-netex] :as row}]
            (let [data-content (if (nil? data-content)
                                 (::t-service/url external-interface)
                                 (parse-content-value data-content))]
              [:div {:key (str i "-" id)}
-              [common-ui/information-row-default data-content [:span (str/join format) [gtfs-viewer-link row]]]]))
+              [common-ui/information-row-default data-content [:span (str/join format)
```

Comment:
Should than when wrap to str?
Like: (str (str/join format) (when url-ote-netex ", NeTEx"))

Reply:
Ok